test-five/react-app-cdk/.github/workflows/cdk-deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: CDK Deploy
+
+on:
+  push:
+    
+    
+
+jobs:
+  deploy:
+    # if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build app 
+      run: |
+        # Change to app directory
+        cd react-app-cdk/my-app
+
+        # Build app
+        npm run build
+
+    - name: Configure aws credentials
+      uses: aws-actions/configure-aws-credentials@master
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
+        aws-region: 'ap-southeast-2'
+
+    - name: Synth stack
+      run: cdk synth
```

Comment:
Do you need to run synth here?

Reply:
perhaps it needs to run after cd to create-react-app, aws documentation states that running synth before cdk deploy is good practice